### DISCUSSION TITLE
Using fetch with tRPC?

### DISCUSSION THREAD

**Discussion by conduongtong** [trpc/trpc#5627] [Q&A]
So basically I have a backend but not using tRPC. In client, I want to use Next and tRPC, should I fetch it directly like normal or through procedure of tRPC?

I can see NextJS documentation using API route:
```ts
export async function GET(request: Request) {
  const { searchParams } = new URL(request.url)
  const id = searchParams.get('id')
  const res = await fetch(`https://data.mongodb-api.com/product/${id}`, {
    headers: {
      'Content-Type': 'application/json',
      'API-Key': process.env.DATA_API_KEY!,
    },
  })
  const product = await res.json()
 
  return Response.json({ product })
}
```
How about procedure (same as API route), is that normal to call an Backend API within an Frontend API?

**Comment by RasenGUY** [comment]
if this is still open, have you experienced cookies from external fetch calls ?
